xpdtools/tools.py

```diff
@@ -581,3 +583,85 @@ def inner(x, *args, **kwargs):
         return func(*args, **kwargs)
 
     return inner
+
+
+def max_value(g, r):
+    """Returns largest value
+
+    Parameters
+    ----------
+    g : ndarray g(r) of the pdf
+    r : ndarray of corresponding r values
+
+    Returns
+    -------
+    float :
+        Maximum g value
+
+    """
+    return np.amax(g)
+
+
+def tallest_peak(g,r):
+    """Returns r,g(r) of the tallest peak
+
+       Parameters
+       ----------
+       g : ndarray g(r) of the pdf
+       r : ndarray of corresponding r values
+
+       Returns
+       -------
+       float :
+           r value of the tallest peak
+        float :
+            g(r) value of the tallest peak
+
+       """
+    peaks = sig.find_peaks(g)
+    height = []
+    r_val = []
+    for i in peaks[0]:
+        height.append(g[i])
+        r_val.append(r[i])
+    return r_val[np.argmax(height)], np.amax(height)
+
+
+def total_counts(g,r):
+    """Returns total number of data points in graph
+
+           Parameters
+           ----------
+           g : ndarray g(r) of the pdf
+           r : ndarray of corresponding r values
+
+           Returns
+           -------
+           float :
+               total number of data points
+
+
+           """
+    return len(g)
+
+
+def average_pearson(group, g,r):
+    """Computes the average pearson of this PDF with the rest of the group
+
+               Parameters
```

Comment:
This has too many indents. `Parameters` should be on the same indentation as the description line.